rgw/v2/tests/s3_swift/configs/test_bucket_lc_object_exp_multipart.yaml

```diff
@@ -0,0 +1,15 @@
+# test_bucket_lc_object_exp_multipart.py
+config:
+  user_count: 1
+  bucket_count: 1
+  objects_count: 1000
```

Comment:
Since we re writing 1K objects, I would request you to use `local_file_delete = true` parameter